doc/classes/TranslationServer.xml

```diff
@@ -60,6 +60,29 @@
 				It will return a [code]nullptr[/code] if there is no [Translation] instance that matches the [code]locale[/code].
 			</description>
 		</method>
+		<method name="is_pseudolocalization_enabled" qualifiers="const">
+			<return type="bool">
+			</return>
+			<description>
+				Returns a boolean suggesting whether Pseudolocalization is enabled.
```

Comment:
```suggestion
				Returns [code]true[/code] if pseudolocalization is enabled, [code]false[/code] otherwise. See also [member ProjectSettings.internationalization/pseudolocalization/use_pseudolocalization].
```